airbyte-integrations/connectors/source-close-com/source_close_com/manifest.yaml

```diff
@@ -194,9 +194,10 @@ definitions:
       name: "email_thread_activities"
       path: "activity/emailthread"
   email_activities_stream:
-    $ref: "#/definitions/activities_base_stream"
+    $ref: "#/definitions/base_stream"
```

Comment:
how disabled pagination can help with limiting error?

Reply:
@lazebnyi The stream previously used `_skip` parameter for pagination. That was what caused the error. According to the API doc, in such cases, it is better to use the `date_created` field for this. But in airbyte, the `date_created` field is best suited for incremental sync, so the pagination is handled through incremental sync. Makes sense?

Please see here for reference - https://developer.close.com/topics/pagination/#deep-pagination

Reply:
please add this to PR description in How section